index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
+  .header_container {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+
+    /* left: 0px;
+    top: 0px; */
+
+    background: #3c3a39;
+    color: red;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .item {
+    align-self: center;
+  }
+
+  .name {
+    font-family: Roboto, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .menu_bar {
+    width: 24px;
+    height: 24px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .menu_bar img {
+    width: 18px;
+    height: 12px;
+    left: 3px;
+    top: 6px;
+  }
```

Comment:
Please apply the margins and paddings according to the Figma template.
Figma design:
<img width="708" alt="Screenshot 2022-12-02 at 2 21 34 AM" src="https://user-images.githubusercontent.com/25887253/205180246-f176666d-a79b-4db1-871b-64210103afc9.png">
<img width="412" alt="Screenshot 2022-12-02 at 2 20 28 AM" src="https://user-images.githubusercontent.com/25887253/205180263-53853a71-040d-44f0-ad83-a4f0313c4020.png">

Your design:
<img width="386" alt="Screenshot 2022-12-02 at 2 25 40 AM" src="https://user-images.githubusercontent.com/25887253/205180372-52d269b2-9c4a-4b05-b317-2bd1aeb06a2a.png">